Resources/Prototypes/_LostParadise/Entities/Clothing/Head/helmet.yml

```
@@ -137,3 +137,169 @@
     sprite: _LostParadise/Clothing/Head/Helmets/NPZ/NPZassault.rsi
   - type: Clothing
     sprite: _LostParadise/Clothing/Head/Helmets/NPZ/NPZassault.rsi
+
+- type: entity
+  parent: ClothingHeadBase
+  id: LPPClothingHeadHelmetHeavy
+  categories: [ HideSpawnMenu ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)